content/en/docs/reference/kubernetes-api/ref/authentication/tokenrequest-v1.md

```diff
@@ -0,0 +1,161 @@
+---
+api_metadata:
+  apiVersion: "authentication.k8s.io/v1"
+  import: "k8s.io/api/authentication/v1"
+  kind: "TokenRequest"
+collapsible: false
+content_type: "api_reference"
+description: "TokenRequest requests a token for a given service account."
+draft: false
+title: "TokenRequest"
+weight: 2
+---
+
+`apiVersion: authentication.k8s.io/v1`
+
+`import "k8s.io/api/authentication/v1"`
+
+
+## TokenRequest
+
+TokenRequest requests a token for a given service account.
+
+
+- **apiVersion**: authentication.k8s.io/v1
+
+
+- **kind**: TokenRequest
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/objectmeta-#objectmeta" >}}">ObjectMeta</a>)
+
+
+- **spec** (<a href="{{< ref "../authentication/tokenrequest-v1#tokenrequestspec" >}}">TokenRequestSpec</a>), required
+
+
+- **status** (<a href="{{< ref "../authentication/tokenrequest-v1#tokenrequeststatus" >}}">TokenRequestStatus</a>)
+
+
+
+
+
+
+## TokenRequestSpec
+
+TokenRequestSpec contains client provided parameters of a token request.
+
+
+- **audiences** ([]string), required
+
+  Audiences are the intendend audiences of the token. A recipient of a token must identitfy themself with an identifier in the list of audiences of the token, and otherwise should reject the token. A token issued for multiple audiences may be used to authenticate against any of the audiences listed but implies a high degree of trust between the target audiences.
```

Comment:
nit: How many chars fill this line before the carriage return?

Reply:
For what it's worth, long single lines of descriptive text make PR edits easier.